tests/test_web_protocol.py

```diff
@@ -835,4 +835,4 @@ def test_data_received_force_close(srv) -> None:
     writer.write(b"x")
     writer.close()
     await asyncio.sleep(0.1)
-    logger.debug.assert_called_with('Ignored premature client disconnection 2')
+    logger.debug.assert_called_with('Ignored premature client disconnection.')
```

Comment:
How is this related to the contextvars change?

Reply:
This fixed tests, because I extracted some logic into `_handle_request`

Reply:
see https://travis-ci.com/aio-libs/aiohttp/jobs/213717795#L3044